urbackupserver/www/translations/urbackup.webinterface/it_IT.po

```diff
@@ -808,7 +808,7 @@ msgid "tDo image backups over internet"
 msgstr "Esegui backup immagine attraverso Internet"
 
 msgid "tDo full file backups over internet"
-msgstr "Esegui backup file atraverso Internet"
+msgstr "Esegui backup file attraverso Internet"
```

Comment:
Changes here will be overwritten once the files are updated from Transifex

Reply:
hah ok, so where should I do this change?